tests/functionnal/Session.php

```diff
@@ -41,13 +41,13 @@ public function testAddMessageAfterRedirect() {
       $warn_msg = 'There was a warning. Be carefull.';
       $info_msg = 'All goes well. Or not... Who knows ;)';
 
-      $this->array($_SESSION['MESSAGE_AFTER_REDIRECT'])->isEmpty();
+      $this->array($_SESSION)->notHasKey('MESSAGE_AFTER_REDIRECT');
```

Comment:
This is not true, html class sets an empty array; we should not chnage Html class right now

Reply:
was part of #8423